### PR TITLE
Javalab: code review UI

### PR DESCRIPTION
**Pull request by bencodeorg**
Basic UI components for Javalab code review feature -- a `Comment `component to render a single comment, and a `CommentEditor` component to create a new comment and submit it.

image

## Testing story

Added unit tests for behavior added in this PR (eg, different background colors on comments based on who the comment is from). More tests to come as this gets hooked up to the back end.

## Follow-up work

The UI isn't quite perfectly lined up with [the Figma mocks](https://www.figma.com/file/fZo4fcZmDUioOCaz04Gza4/Javalab-Design-Spec?node-id=405%3A1544), so probably some additional work to get things to look juust right (eg, date text overflows when a comment is from a teacher). I also did not include the menu that will be shown when a user clicks on the ellipsis yet.

I'll also delete the demo comments as I start working on getting this to work with the back end.

### CODE REVIEW

File: apps/src/templates/instructions/ReviewTab.jsx [hunk 1]
```diff
@@ -1,7 +1,16 @@
 import React, {Component} from 'react';
+//import PropTypes from 'prop-types';
```
Comment (molly-moen) [inline apps/src/templates/instructions/ReviewTab.jsx:2]: nit: remove this?

File: apps/src/templates/instructions/codeReview/Comment.jsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  },
+  comment: {
+    clear: 'both',
+    backgroundColor: color.lighter_gray,
+    padding: '10px 12px'
+  },
+  commentContainer: {
+    margin: '0 0 25px 0'
+  },
+  projectOwnerComment: {
```
Comment (molly-moen) [inline apps/src/templates/instructions/codeReview/Comment.jsx]: I think we want the background to be blue when the comment is from 'you' (so if you are a peer/teacher and are looking at your comment on someone else's project, or if it's your own project and your own comment). Is that right @moneppo?

File: apps/src/templates/instructions/codeReview/Comment.jsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    showEllipsisMenu: false
+  };
+
+  renderName = () => {
+    const {name, isFromTeacher} = this.props.comment;
+
+    const teacherCommentSuffix = ` (${javalabMsg.onlyVisibleToYou()})`;
+    return (
+      <span>
+        <span style={styles.name}>{name}</span>
```
Comment (molly-moen) [inline apps/src/templates/instructions/codeReview/Comment.jsx:26]: per the figma I think we want this to be 'you' if it is your comment

File: apps/src/templates/instructions/codeReview/Comment.jsx (hunk 2 above)
Comment (bencodeorg) [reply apps/src/templates/instructions/codeReview/Comment.jsx]: Ooh that would make sense!

**Review by molly-moen — APPROVED** [review]
LGTM!

File: apps/src/templates/instructions/ReviewTab.jsx (hunk 1 above)
Comment (bencodeorg) [reply apps/src/templates/instructions/ReviewTab.jsx:2]: It's going to be re-added when I start including real comments as props, which is imminently 🤞

**Pull request merged**